README.md

```diff
@@ -0,0 +1,13 @@
+# `strong-config`
+
+Secret management for NodeJS. Based on [SOPS](https://github.com/mozilla/sops).
+
+[![pipeline status](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/pipeline.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+
+[![coverage report](https://git.brickblock.sh/devops/strong-config-ts-mirror/badges/master/coverage.svg)](https://git.brickblock.sh/devops/strong-config-ts-mirror/commits/master)
+
+## How-to
+
+Set the environment variable `RUNTIME_ENVIRONMENT`
```

Comment:
Can we improve this, so it's easier for someone to clone the project and start developing?

1. Prerequisites e.g sops must be installed => google kms => account on google cloud with permissions to x, y, z 
2. How to start developing 
3. How to test
4. How to use

Reply:
Agree but I'd say this is a separate MR.

(@mohoff these are the types of comments one typically avoids with a clearer MR description)

Reply:
Should probably also warrant its own ticket, our README needs to be ace. If we want people to adopt this, the README is probably their first touchpoint.

Reply:
Definitely will do that!

But not related to `Migrate from Flow to Typescript`. Would appreciate a ticket @PaddyMc

Reply:
Ace, just created two issues :)